src/main/java/fuzz/Fuzz.java

```
@@ -23,33 +22,31 @@ public static void FileIO(){
             String[] badWords = new String[400];
 
             while ((line = bufferedReader.readLine()) != null) {
-                //  System.out.println(line);
                 for (int i = 0; i < badWords.length; i++) {
                     badWords[i] = line;
+                    //  System.out.println(badWords[i]);
                 }
             }
             reader.close();
+            return badWords;
+
         } catch (Exception e) {
             e.printStackTrace();
```

This should likely be a runtime exception rather than a sprint statement

---

Sorry this comment wasn't to change the type of exception being caught, I meant that you should throw the exception to just kill the program here rather than printing and continuing